frontstage/__init__.py

```diff
@@ -63,3 +64,10 @@ def inject_availability_message():
 
 # Bind routes to app
 import frontstage.views  # NOQA
+
+
+@app.before_request
+def redirect_to_maintenance_page():
+    under_maintenance = True
```

Comment:
So this PR, if merged right now, would just endlessly render that one template and disable all the other functionality.... seriously?
This clearly needs more work doing to it to make it configurable and a test to show the result with it switched on